src/Common/src/System/Net/Security/Unix/SafeDeleteNegoContext.cs

```diff
@@ -12,10 +12,24 @@ namespace System.Net.Security
 {
     internal sealed class SafeDeleteNegoContext : SafeDeleteContext
     {
+        private SafeGssCredHandle _acceptorCredential;
         private SafeGssNameHandle _targetName;
         private SafeGssContextHandle _context;
         private bool _isNtlmUsed;
 
+        public SafeGssCredHandle AcceptorCredential
+        {
+            get
+            {
+                if (_acceptorCredential == null)
+                {
+                    _acceptorCredential = SafeGssCredHandle.CreateAcceptor();
+                }
```

Comment:
This if can be collapsed into `_acceptorCredential ??= SafeGssCredHandle.CreateAcceptor();`

Reply:
Thanks. I forgot about the null coalescing operator [assignment](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator#examples) form.